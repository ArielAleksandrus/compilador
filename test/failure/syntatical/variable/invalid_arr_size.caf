int func1(){
    int a[3+];
    retorne 0;
}
programa{}

