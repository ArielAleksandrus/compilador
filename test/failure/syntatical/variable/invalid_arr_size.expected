ERROR => Line 2, col 12: Not a valid expression
